images_config/kickstart/kickstart-robot.sh

```diff
@@ -0,0 +1,465 @@
+        echo "found GRUB_RECORD_FAIL flag already, skipping SetupGrubRecFail (update-grub call)"
+    else
+        echo "GRUB_RECORDFAIL_TIMEOUT=2" >> /etc/default/grub
+        update-grub
+    fi
+}
+
+#TODO: requires user input - No working solution found so far
+function KeyboardLayout {
+    printHeader "KeyboardLayout"
+    L='de' && sed -i 's/XKBLAYOUT=\"\w*"/XKBLAYOUT=\"'$L'\"/g' /etc/default/keyboard
+    apt-get install console-data -y -f
+    dpkg-reconfigure keyboard-configuration
+}
+
+function ConfigureSSH {
+    printHeader "ConfigureSSH"
+    apt-get install openssh-server -y
+
+    if grep -q "X11Forwarding" /etc/ssh/sshd_config; then
+        sed -i 's/X11Forwarding.*$/X11Forwarding yes/g' /etc/ssh/sshd_config
+    else
+        echo "X11Forwarding yes" >> /etc/ssh/sshd_config
+    fi
+    if grep -q "X11UseLocalhost" /etc/ssh/sshd_config; then
+        sed -i 's/X11UseLocalhost.*$/X11UseLocalhost no/g' /etc/ssh/sshd_config
+    else
+        echo "X11UseLocalhost no" >> /etc/ssh/sshd_config
+    fi
+    if grep -q "PermitRootLogin" /etc/ssh/sshd_config; then
+        sed -i 's/PermitRootLogin.*$/PermitRootLogin yes/g' /etc/ssh/sshd_config
+    else
+        echo "PermitRootLogin yes">> /etc/ssh/sshd_config
+    fi
+    if grep -q "ClientAliveInterval" /etc/ssh/sshd_config; then
+        sed -i 's/ClientAliveInterval.*$/ClientAliveInterval 60/g' /etc/ssh/sshd_config
+    else
+        echo "ClientAliveInterval 60" >> /etc/ssh/sshd_config
+    fi
+    
+    systemctl restart ssh
+}
+
+function ChronySetup {
+    printHeader "ChronySetup"
+    apt-get install chrony -y -f
+    if [ "$INSTALL_TYPE" == "master" ]; then
+        wget -O /etc/chrony/chrony.conf https://raw.githubusercontent.com/ipa320/setup_cob4/master/cob-pcs/chrony_server
+
+    elif [ "$INSTALL_TYPE" == "slave" ]; then
+        HOSTNAME=$(cat /etc/hostname)
+        SERVERNAME=$(echo ${HOSTNAME%-*}-b1)
+        wget -O /etc/chrony/chrony.conf https://raw.githubusercontent.com/ipa320/setup_cob4/master/cob-pcs/chrony_client
+        sed -i "s/server_ip/${SERVERNAME}/g" /etc/chrony/chrony.conf
+    fi
+}
+
+#udev rules
+function SetupUdevRules {
+    printHeader "SetupUdevRules"
+    wget -O /etc/udev/rules.d/98-led.rules https://raw.githubusercontent.com/ipa320/setup_cob4/master/udev_rules/98-led.rules
+    if [ "$INSTALL_TYPE" == "master" ]; then
+            wget -O /etc/init.d/udev_cob.sh https://raw.githubusercontent.com/ipa320/setup_cob4/master/udev_rules/udev_cob.sh
+            chmod +x /etc/init.d/udev_cob.sh
+            update-rc.d udev_cob.sh defaults
+    elif [ "$INSTALL_TYPE" == "slave" ]; then
+        wget -O /etc/udev/rules.d/99-gripper.rules https://raw.githubusercontent.com/ipa320/setup_cob4/master/udev_rules/99-gripper.rules
+    fi
+}
+
+function InstallGitLFS {
+    printHeader "InstallGitLFS"
+    curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
+    apt-get install git-lfs
```

Comment:
there is an additional step for git lfs to work (https://github.com/git-lfs/git-lfs/wiki/Installation)
`git lfs install`

....we need to make sure this is called before we start cloning lfs-enabled repos!